js/src/controls/SplitMapControl.js

```diff
@@ -20,30 +20,42 @@ var LeafletSplitMapControlModel = LeafletControlModel.extend({
     })
 });
 
+function asArray (arg) {
+  return (arg === 'undefined') ? [] : Array.isArray(arg) ? arg : [arg]
+}
+
 var LeafletSplitMapControlView = LeafletControlView.extend({
   initialize: function (parameters) {
         LeafletSplitMapControlView.__super__.initialize.apply(this, arguments);
         this.map_view = this.options.map_view;
     },
 
     create_obj: function () {
-        var left_model = this.model.get('left_layer');
-        var right_model = this.model.get('right_layer');
+        var left_models = this.model.get('left_layer');
+        var right_models = this.model.get('right_layer');
         var layersModel = this.map_view.model.get('layers');
-        layersModel.push(left_model, right_model);
+        layersModel = layersModel.concat(left_models, right_models);
+
+        if (!Array.isArray(left_models)){
```

Comment:
You already do this check in `asArray`, no need to do it twice. Just `left_models = asArray(left_models)` is enough.